### PR TITLE
fix: remove use of `npm bin` (not supported in npm@9.x, default in node-18.14 or greater)

### LINKED ISSUE #39: `npm bin` has been removed in Node 18
[npm bin has been removed with npm 9.0.0](https://github.blog/changelog/2022-10-24-npm-v9-0-0-released/#%E2%9A%A0%EF%B8%8F-notable-breaking-changes), which is what GitHub uses in actions for running Node 18. 

This means the [before each check for the Node executable path](https://github.com/adobe/asset-compute-integration-tests/blob/main/test/index.test.js#L55) will no longer work when this repo is updated to run tests using Node 18.

Solution could potentially be to use `npm list -g` and parse the output.

### PR DESCRIPTION
**Pull request by shazron**
Fixes #39 
Fixes #40

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [X] My code follows the code style of this project.
- [X] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.
- [X] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by pheenomenon — APPROVED** [review]

**Review by tmathern — APPROVED** [review]

**Comment by shazron** [comment]
Can I merge this? Thanks!

File: .github/workflows/node.js.yml [hunk 1]
```diff
@@ -15,7 +15,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [14.x, 16.x]
+        node-version: [14.x, 16.x, 18.x, 20.x]
```
Comment (tmathern) [inline .github/workflows/node.js.yml:18]: Question: I saw the PR from https://github.com/adobe/asset-compute-integration-tests/pull/42. 

Doesn't this mean older node versions become unsupported?

**Review by tmathern — CHANGES_REQUESTED** [review]

File: .github/workflows/node.js.yml (hunk 1 above)
Comment (shazron) [reply .github/workflows/node.js.yml]: @tmathern removed node-14 and 16 which are EOL 1b68606ff1fbb34f2716460145f71885a79ecfb3
#42 was updated as well.

**Review by tmathern — COMMENTED** [review]
Do we send update notices/are our customers aware when we do that?
To make sure their workers don't break?

**Comment by shazron** [comment]
> Do we send update notices/are our customers aware when we do that? To make sure their workers don't break?

Here are the caveats, only for when they create new workers: https://github.com/adobe/generator-app-asset-compute/issues/46

The older unsupported cli versions did not have the `--template` flag (see the removed tests in #42) so they can't use the workaround.

**Review by tmathern — APPROVED** [review]

**Comment by shazron** [comment]
Thanks for the reviews @tmathern, @pheenomenon  and @badvision ! The tests now all pass, could you please merge this in. 🙏

**Comment by tmathern** [comment]
@shazron Merged!

**Pull request merged**